test/k8s-integration/cluster.go

```
@@ -19,9 +19,19 @@ func clusterDownGCE(k8sDir string) error {
 	return nil
 }
 
-func clusterDownGKE(gceZone string) error {
+func clusterDownGKE(gceZone, gceRegion string) error {
+	var locationArg, locationVal string
+	switch {
+	case len(gceZone) > 0:
```

do we want to throw an error if both/neither are specified